stratum/CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ set(STRATUM_INCLUDES
     ${PROTO_OUT_DIR}
 )
 
+if(DPDK_TARGET)
```

Comment:
1. The symbol name should be in curly braces (`${KRNLMON_SOURCE_DIR}`).
2. Modern CMake calls for applying properties to individual objects rather than specifying them globally.

Reply:
Done